src/Domain.Tests/Apprenticeship/WhenSettingPaymentStatus.cs

```diff
@@ -42,14 +36,45 @@ public void And_NewStatusIsSameAsOld_Then_Throws()
     public void And_NewStatusIsFrozen_Then_PaymentsAreFrozen()
     {
         //Arrange
+        var userId = _fixture.Create<string>();
+        var timeChanged = DateTime.Now;
         var apprenticeship = CreateApprenticeshipDomainModel();
 
         //Act 
-        apprenticeship.SetPaymentStatus(true, _userId, DateTime.Now);
+        apprenticeship.SetPaymentStatus(true, userId, timeChanged);
```

Comment:
Similar to our conversation about the web controller yesterday, is it worth updating this to be "SetPaymentsFrozen" so it's super clear that true means frozen from the method name?